source/ed/connectors/geopal_parser.cpp

```diff
@@ -107,21 +107,17 @@ void GeopalParser::fill_nodes(){
         }
         int insee_c = reader.get_pos_col("code_insee");
         int code_post_c = reader.get_pos_col("code_post");
-        int x_c = reader.get_pos_col("x_adresse");
-        int y_c = reader.get_pos_col("y_adresse");
         while(!reader.eof()){
             std::vector<std::string> row = reader.next();
-            std::string uri;
-            if (reader.is_valid(x_c, row) && reader.is_valid(y_c, row) && reader.is_valid(insee_c, row)){
-                uri = row[x_c] + row[y_c];
+            if (reader.is_valid(code_post_c, row) && reader.is_valid(insee_c, row)){
                 auto adm = this->data.admins.find(row[insee_c]);
                 if (adm != this->data.admins.end()){
                     if(reader.is_valid(code_post_c, row)){
-                        adm->second->postcode = row[code_post_c];
+                        if(adm->second->postcode.empty()){
+                            adm->second->postcode = row[code_post_c];
```

Comment:
so we take the first one for postcode ? 

I think we can do like in city and take a range of postal code no ?